community/values/src/main/java/org/neo4j/values/storable/Values.java

```diff
@@ -48,7 +48,7 @@
     public static final Value MAX_STRING = Values.booleanValue( false );
     public static final BooleanValue TRUE = Values.booleanValue( true );
     public static final BooleanValue FALSE = Values.booleanValue( false );
-    public static final TextValue EMPTY_STRING = Values.stringValue( "" );
```

Comment:
Interesting...

Really `Values.stringValue( value )` should be made to return `StringValue.EMPTY` if `value.isEmpty()`.

But that's for another changeset, and this is a good change in any case.